README.md

```diff
@@ -37,6 +37,18 @@ lineage:
     remote-url: https://github.com/felddy/extra-skel-sauce.git
 ```
 
+### Configure a [cisagov] repository for automatic Lineage updates ###
+
+If you are a [cisagov] repository maintainer and wish for your repository to
+receive Lineage updates you must do the following:
+
+1. Ensure your project is properly based on a [Skeleton repository](https://github.com/cisagov/development-guide/blob/develop/project_setup/skeleton-list.md)
```

Comment:
Wording/capitalization suggestion:
```suggestion
1. Ensure your project is properly based on a [cisagov skeleton repository](https://github.com/cisagov/development-guide/blob/develop/project_setup/skeleton-list.md)
```

Reply:
I think I would prefer to leave the `cisagov` out simply because the configuration could be such that any other public repository could be used as the parent repository. I only link to the `cisagov` skeletons as an example.

Reply:
After thinking about it I decided to update the wording a bit. Please see [45ca44b](https://github.com/cisagov/action-lineage/pull/66/commits/45ca44ba74c8dfe3d724290e7432e2fa52e4712e) and let me know your thoughts.

Reply:
I like it! 👍